tests/testAutopas/tests/autopasInterface/Newton3OnOffTest.cpp

```diff
@@ -17,11 +17,10 @@ TEST_P(Newton3OnOffTest, countFunctorCallsTest) {
   auto contTravStr = std::get<0>(GetParam());
   auto dataLayoutStr = std::get<1>(GetParam());
 
-  transform(contTravStr.begin(), contTravStr.end(), contTravStr.begin(), ::tolower);
-  transform(dataLayoutStr.begin(), dataLayoutStr.end(), dataLayoutStr.begin(), ::tolower);
-  auto containerOption = autopas::utils::StringUtils::parseContainerOptions(contTravStr).begin().operator*();
-  auto traversalOption = autopas::utils::StringUtils::parseTraversalOptions(contTravStr).begin().operator*();
-  auto dataLayoutOption = autopas::utils::StringUtils::parseDataLayout(dataLayoutStr).begin().operator*();
+  auto contTravStrVector = autopas::utils::StringUtils::tokenize(contTravStr, "+");
```

Comment:
make this unneeded, by using the actual options and not the string representations inside of the parametrization

Reply:
```
::testing::WithParamInterface<std::tuple<std::string, std::string>>
```
will become sth. like:
```
::testing::WithParamInterface<std::tuple<ContainerOption, TraversalOption>>
```